go/vt/vtadmin/api_test.go

```diff
@@ -42,6 +42,7 @@ import (
 	"vitess.io/vitess/go/vt/vttablet/tmclient"
 
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/proto/vtadmin"
```

Comment:
double import here; vscode gets me with this one all the time

Reply:
Thanks for catching that